bootloader/prebootloader/src/rtl872x/part1/sleep_handler.cpp

```diff
@@ -249,8 +249,31 @@ void configureDeepSleepWakeupSource(const hal_sleep_config_t* config) {
 
 // Copy and paste from SOCPS_DeepSleep_RAM()
 void enterDeepSleep() {
+#if PLATFORM_ID == PLATFORM_P2
+    // dirty-hack for Photon2 D7: PA27 (SWD-DIO)
+    uint32_t bitMask = 1 << 27;
```

Comment:
Are there other pins at risk of this behavior or is this just because of how the SWD pin is configured in the SDK?